frontend/src/layout/header/Search.tsx

```diff
@@ -80,7 +80,7 @@ export const SearchField: React.FC<SearchFieldProps> = ({ variant }) => {
             clearTimeout(lastTimeout.current);
             search(currentRef(ref).value);
         }}>
-            <label>
+            <label css={{ display: "flex" }}>
```

Comment:
I think you can remove the `max-width` on the `<input>` now.

Reply:
Hmm correct, I guess then we won't need the `extraCss` thing anymore either and can move that stuff into the containing div.